skylog/storage/gcp/pubsub/main.go

```diff
@@ -28,13 +28,15 @@ import (
 )
 
 var (
-	projectID  = flag.String("project", "skylog-test", "The GCP project ID")
-	psTopic    = flag.String("topic", "build-jobs", "The Pub/Sub topic for build jobs")
-	treeID     = flag.Int64("tree_id", 1, "The ID of the tree under construction")
-	beginIndex = flag.Int64("begin", 0, "The beginning of the tree building range")
-	endIndex   = flag.Int64("end", 0, "The ending of the tree building range")
-	maxJobSize = flag.Int("job_size", 256, "The maximal number of entries in a build job")
-	maxRate    = flag.Float64("rate", 100000, "The average rate of adding entries per second")
+	projectID   = flag.String("project", "skylog-test", "The GCP project ID")
+	psTopic     = flag.String("topic", "build-jobs", "The Pub/Sub topic for build jobs")
```

Comment:
How about `buildJobsTopic`/`build_jobs_topic` for the var and flag name?

Reply:
Ack. Let me address this comment and the one below in a separate PR, as it's orthogonal.

Reply:
#1710